### PR TITLE
fix(slack): handlebars error handling

### PR DESCRIPTION
**Pull request by Gauravudia**
## Description of the change

> Fixing bugsnag error for slack destination

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

>  https://app.bugsnag.com/rudderlabs/rudder-transformer-error-reporter/errors/63ff7496326a910009c3d87e?event_id=63ff749600b8574b79810000&i=wh&m=nw

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/v0/destinations/slack/transform.js [hunk 1]
```diff
@@ -208,7 +208,13 @@ const processTrack = (message, destination) => {
 
   logger.debug('templateInputTrack: ', templateInput);
 
-  const resultText = eventTemplate(templateInput);
+  let resultText;
+  try {
+    resultText = eventTemplate(templateInput);
+  } catch (err) {
+    throw new ConfigurationError('Something is wrong with the event template');
```
Comment (saikumarrs) [inline src/v0/destinations/slack/transform.js:215]: Include the template in the error message.

Comment (saikumarrs) [reply src/v0/destinations/slack/transform.js:215]: Please add the original error message if it can help.

Comment (Gauravudia) [reply src/v0/destinations/slack/transform.js]: - template added.
- the original message is not very informative `(lookupProperty(...) || (depth0 && lookupProperty(...)) || alias2).call is not a function`

File: src/v0/destinations/slack/transform.js [hunk 2]
```diff
@@ -212,7 +211,7 @@ const processTrack = (message, destination) => {
   try {
     resultText = eventTemplate(templateInput);
   } catch (err) {
-    throw new ConfigurationError('Something is wrong with the event template');
+    throw new ConfigurationError(`Something is wrong with the event template: ${template}`);
```
[the remark this answers is not in the document]
Comment (Gauravudia) [reply src/v0/destinations/slack/transform.js]: added

**Review by saikumarrs — DISMISSED** [review]

**Review by ItsSudip — DISMISSED** [review]

**Review by ItsSudip — APPROVED** [review]

**Review by mihir-4116 — APPROVED** [review]

**Pull request merged**